contracts/core/StandardizedYield/implementations/Adapter/PendleInfiniFiAdapter.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.17;
+
+import "../../../../interfaces/IStandardizedYieldAdapter.sol";
+import "../../../../interfaces/InfiniFi/InfiniFiInterfaces.sol";
+
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+
+contract PendleEmptyAdapter is IStandardizedYieldAdapter {
+    using SafeERC20 for IERC20;
+
+    error TokenMustBeUSDC();
+
+    address public constant USDC = 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
+    address public constant IUSD = 0x48f9e38f3070AD8945DFEae3FA70987722E3D89c;
+
+    address public constant PIVOT_TOKEN = IUSD;
+
+    address public constant GATEWAY_PROXY = 0x3f04b65Ddbd87f9CE0A2e7Eb24d80e7fb87625b5;
+    address public constant MINT_CONTROLLER = 0x49877d937B9a00d50557bdC3D87287b5c3a4C256;
+    address public constant REDEEM_CONTROLLER = 0xCb1747E89a43DEdcF4A2b831a0D94859EFeC7601;
```

Comment:
Read these dynamically, we want to be able to upgrade this part of the protocol without breaking Pendle integration :
 - gateway.getAddress("mintController")
 - gateway.getAddress("redeemController")